lib/state/action-types.ts

```diff
@@ -63,6 +64,7 @@ export type RemoteNoteUpdate = Action<
   { noteId: T.EntityId; data: T.Note }
 >;
 export type RestoreNote = Action<'RESTORE_NOTE', { previousIndex: number }>;
+export type ShowDialog = Action<'SHOW_DIALOG', { dialog: string }>;
```

Comment:
```ts
{ dialog: T.DialogType }
```

Reply:
THANK YOU lol I looked for it and couldn't find the one I missed!